python/cli/prelude_cli/views/shared.py

```diff
@@ -1,15 +1,19 @@
-import click
-
 from functools import wraps
+from rich import print_json
 from rich.progress import Progress, TextColumn, SpinnerColumn
 
-def handle_api_error(func):
+
+def pretty_print(func):
     @wraps(func)
     def handler(*args, **kwargs):
         try:
-            return func(*args, **kwargs)
+            res = func(*args, **kwargs)
+            msg = None
+            if isinstance(res, tuple):
+                res, msg = res
+            return print_json(data=dict(status='complete', result=res, message=msg))
```

Comment:
> it could be a single string or a bool even
it's on us to make dicts where there are none (ex. the natural API response of register_endpoint is a string, but I convert to a dict before returning into this decorator)

> Did you consider pluralizing it
Definitely can if you'd like / if that's normal

Reply:
Normal? I'm not sure what normal is. I see lots of json responses that make sure it's always a list because you never know, and then I see some that aren't. If the argument for this consistent wrapper is for consumability, I feel like outputting a dict vs a list is counter to that....but maybe i'm overthinking it.

Reply:
okee done